specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1203,17 +1217,15 @@ components:
       properties:
         typeGerechtigde:
           type: "string"
-          description: "Het type zakelijk recht dat deze gerechtigde heeft. Komt overeen met AardzakelijkRecht"
+          description: "Het type zakelijk recht dat deze gerechtigde heeft."
         aanvangsdatum:
           type: "string"
           description: "Datum waarop het zakelijk recht van kracht is geworden. "
           format: "date"
         erfpachtCanon:
           $ref: "#/components/schemas/ErfpachtCanon"
         tenaamstellingen:
-            type: "array"
-            items:
-              $ref: "#/components/schemas/Tenaamstelling"
+          $ref: "#/components/schemas/Tenaamstelling"
```

Comment:
als ik het imKad lees is het een 0..n relatie. Dus een array. Ook heet het attribuut tenaamstellingen (in meervoud), dus verwacht ik er meerdere te kunnen krijgen.

Reply:
Maar dit is nu een resource die wij zakelijk gerechtigde hebben genoemd. Dat betekent dat deze resource (die wij bedacht hebben) een persoon is die een specifieke tenaamstelling heeft en een daarbijbehorend recht heeft. Ik snap dat een zakelijk recht meerdere tennaamstellingen kan hebben die ieder weer aan een persoon gekoppeld zijn.  Maar voor deze resource moet je de relaties vanuit de persoon bekijken.

Reply:
duidelijk, mee eens. Zie #293 want ik denk dat de zakelijkgerechtigde nu vreemd is gemodelleerd. heb dat voorgelegd aan Hilde